src/sbt-test/sbt-freestyle-protogen/with-sbt-freestyle/project/plugins.sbt

```diff
@@ -0,0 +1,2 @@
+addSbtPlugin("io.frees" % "sbt-frees-protogen" % sys.props("plugin.version"))
+addSbtPlugin("io.frees" % "sbt-freestyle" % "0.3.7")
```

Comment:
This will also need to be bumped when `./plugins.sbt` has its `sbt-freestyle` bumped, which is a little annoying.